src/js/providers/html5.js

```diff
@@ -524,7 +524,7 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
             });
         }
         if (_levels.length && _levels[0].type !== 'hls') {
-            _this.sendMediaType(_levels);
```

Comment:
this outsourced `mediaType` determination to the default provider, this change makes it the responsibility of the html5 provider andd removes a gating that required `hls` for the html5 provider to make the call